### ISSUE TITLE
Add Traccar support

### ISSUE THREAD

**Issue by brian7704** [brian7704/OpenTAKServer#19]
Connect to Traccar's websocket and save the tracker data it sends. Also add an API route to accept data from the Android and iOS Traccar clients directly.

**Comment by nerdalertdk** [comment]
Would Meshtastic not make this "redundant" ?

**Comment by brian7704** [comment]
No, [Traccar](https://www.traccar.org/) is different from Meshtastic. It receives location updates from [tracking devices](https://freematics.com/store/index.php?route=product/product&product_id=85&tracking=5fbca3beebc8d) that are usually installed in vehicles. It is mainly used for vehicle fleet management, and it has some other use cases that TAK users might benefit from.

Unlike Meshtastic, Traccar doesn't offer any chat support or data transmission other than location updates.

**Comment by brian7704** [comment]
This should be a plugin
